config/sync/system.site.yml

```diff
@@ -5,7 +5,7 @@ slogan: ''
 page:
   403: ''
   404: ''
-  front: /user/login
+  front: /node/1
```

Comment:
This change is not related to media and should not be a part of this pull request.